.gitignore

```diff
@@ -2,3 +2,6 @@
 composer.lock
 composer.phar
 vendor
+tests/coverage
+.phpunit.result.cache
+clover.xml
```

Comment:
These should have went into your global .gitignore.

Reply:
tests/coverage and clover.xml are specified in the phpunit.xml. Are you sure they should be global?

Reply:
I have `.phpunit.result.cache` in pretty much in every other project. Two others are not belong here in the first place. E.g. you can put them into a `build` folder, then ignore it globally. Or add a local `.gitignore` with a `*` in that folder.

Reply:
I agree with you  concerning  `.phpunit.result.cache` but for the 2 others adding a dedicated folder which itself is ignored looks just as adding out of scope complexity. It could be necessary, if ever a full build process had to be configured, but later, when the `build` folder would gather more stuff. And taking this in consideration here seems much less important than considering coverage for example.

If you have an example of a build process applied to other Pear packages, I would follow it gladly. Is it the case?